common/protocols/src/main/proto/app/coronawarn/server/common/protocols/external/exposurenotification/diagnosis_key_batch.proto

```diff
@@ -6,5 +6,20 @@ option java_multiple_files = true;
 import "app/coronawarn/server/common/protocols/external/exposurenotification/temporary_exposure_key_export.proto";
 
 message DiagnosisKeyBatch {
-  repeated TemporaryExposureKey keys = 1;
+  repeated DiagnosisKey keys = 1;
+}
+
+message DiagnosisKey {
+  bytes keyData = 1; // key
+  uint32 rollingStartIntervalNumber = 2;
+  uint32 rollingPeriod = 3; // number of 10-minute windows between key rolling
+  int32 transmissionRiskLevel = 4; // risk of transmission
+  repeated string visitedCountries = 5;
+  string origin = 6; // country of origin
+  VerificationType verificationType = 7; // set by backend
+}
+
+enum VerificationType {
+  LAB_VERIFIED = 0;
+  SELF_REPORTED = 1;
```

Comment:
any ideas how to keep this in sync with https://github.com/eu-federation-gateway-service/efgs-federation-gateway/blob/master/src/main/proto/Efgs.proto ?